morse_decoder.rb

```diff
@@ -0,0 +1,54 @@
+MORSE = {}.freeze
+MORSE = {
```

Comment:
- [ ] [OPTIONAL] You're declaring the `MORSE` variable multiple times, kindly make sure to fix this, so you avoid a `warning` message about initializing a constant

Reply:
Thanks @Ismailco . I will fix and push